tests/e2e/specs/pacbio_run_create.js

```diff
@@ -37,53 +68,48 @@ describe('Pacbio Run Create view', () => {
       .select('Do Not Generate')
       .get('#bindingKitBoxBarcode')
       .type('12345')
-      .get('button')
-      .contains('Update')
+      .get('#updateWellBtn')
       .click()
-    // lots of complex calls. Maybe we only need one.
-    cy.intercept('/v1/pacbio/runs', {
-      statusCode: 201,
-      body: {
-        data: {
-          id: '1',
-          type: 'runs',
-          links: {
-            self: '/v1/pacbio/runs/1',
-          },
-        },
-      },
-    })
-    cy.intercept('/v1/pacbio/runs/plates', {
-      statusCode: 201,
+    cy.get('button')
+      .contains('Create')
+      .click()
+    // TODO: we need a success message.
+  })
+
+  it('creates a run unsuccessfully', () => {
+    cy.intercept('/v1/pacbio/runs/wells', {
+      statusCode: 422,
       body: {
         data: {
-          id: 1,
-          type: 'plates',
-          links: {
-            self: '/v1/pacbio/runs/plates/1',
+          errors: {
+            error1: ['some error'],
           },
         },
       },
     })
-    cy.intercept('/v1/pacbio/runs/wells', {
-      statusCode: 201,
-      body: { data: {} },
-    })
+    cy.intercept('/v1/pacbio/runs/1', {})
+    const dataTransfer = new DataTransfer()
+
+    cy.visit('#/pacbio/runs')
+    cy.get('button')
+      .contains('New Run')
+      .click()
+    cy.get('#sequencing-kit-box-barcode').type('Lxxxxx101826100123199')
+    cy.get('#dna-control-complex-box-barcode').type('Lxxxxx101717600123199')
+    cy.get('#system-name').select('Sequel IIe')
+    cy.get('.list-group-item')
```

Comment:
Similarly here I'd suggest fetching based on label, but it appears that Cypress doesn't actually contain such functionality out the box. Not worth addressing now.